rholang/src/test/scala/coop/rchain/rholang/interpreter/storage/ChargingRSpaceTest.scala

```diff
@@ -341,20 +339,17 @@ class ChargingRSpaceTest extends fixture.FlatSpec with TripleEqualsSupport with
     implicit val s          = implicitly[Sync[Task]]
     val chargingRSpace      = ChargingRSpace.pureRSpace(s, costAlg, pureRSpace)
     try {
-      test(TestFixture(chargingRSpace, costAlg, pureRSpace))
+      test(TestFixture(chargingRSpace, costAlg))
     } finally {
       pureRSpace.close()
     }
   }
-
-  final case class TestFixture(
-      chargingRSpace: ChargingRSpace,
-      costAlg: CostAccounting[Task],
-      pureRSpace: RhoISpace
-  )
```

Comment:
Scalac doesn't cope well with inner classes, this was reporting some error about inner reference. Moving it to `object` fixed this issue.